From 5397025640ea3ba7499e183993e11ede5766ba1d Mon Sep 17 00:00:00 2001
From: Otavio Salvador <otavio@ossystems.com.br>
Date: Wed, 7 Nov 2018 09:45:55 -0200
Subject: [PATCH] Revert "setup.py: Update requests requirement for 2.20.0"
Organization: O.S. Systems Software LTDA.

This reverts commit 3e0e8d56681de7f90192008f0ebee0dfd95e3723.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index f9fd669..fbcfc1d 100644
--- a/setup.py
+++ b/setup.py
@@ -22,7 +22,7 @@ setup(
         'prompt-toolkit>=2.0.0,<3.0.0',
         'pycrypto',
         'updatehub-package-schema>=1.0.2',
-        'requests>=2.20.0',
+        'requests>=2',
         'rfc3987>=1.3',
     ],
     author='O.S. Systems Software LTDA',
-- 
2.19.1

